### ISSUE TITLE
Accessibility (WCAG level A) Skipped header levels on Statistics page for specific packages

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1183]
Go to https://nuget.org/stats/packages/Microsoft.Web.Infrastructure/1.0.0.0?groupby=ClientName and view source
2. Scrutinize how we're using h1..h6 tags to organize headers.

Result: On this page, there is an h2 element, but no h1 elements.
Expected: We shouldn't skip levels when using the h1..h6 tags. In other words, if we're using an h3 tag, that means there should be an h1 and h2 tag somewhere else on the page. In this case, I'd expect us to use h1 tags and re-style the tags as necessary to achieve the visual effect we want.

**Comment by johnataylor** [comment]
We have a new gallery design.
